src/components/Person/Person.jsx

```diff
@@ -1 +1,41 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
```

Comment:
use destructuring according to checklist.md
![image](https://github.com/mate-academy/react_person/assets/91946036/357dc0a2-7eea-4768-8491-31d8411ee30e)
![image](https://github.com/mate-academy/react_person/assets/91946036/693d6006-d6cf-4a44-b117-68dec1ceb027)